tests/acceptance/features/webUIUserLDAP/groupMembership.feature

```diff
@@ -26,7 +26,7 @@ Feature: group membership
     And the administrator adds user "Carol" to ldap group "grp1"
     When the user re-logs in as "Carol" using the webUI
     Then folder "simple-folder (2)" should be listed on the webUI
-    And folder "simple-folder (2)" should be marked as shared with "grp1" by "Alice Hansen" on the webUI
+    And folder "simple-folder (2)" should be marked as shared with "grp1" by "Alice" on the webUI
```

Comment:
@phil-davis aren't we supposed to check for the display name in this step?

Reply:
Since @jasson99 changes in core, we put the username here and the test code underneath looks up the display name of the user and then checks for the display name.